Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 36
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	decrease_zero_inflation
	2

[Wed Nov 11 19:39:03 2020]
rule decrease_zero_inflation:
    input: results/Metadata/SRR12845350.json, results/Metadata/SRR8094770.json, results/Rawdata/HTSeq_count/SRR12845350_genes.tsv, results/Rawdata/HTSeq_count/SRR8094770_genes.tsv, results/Rawdata/HTSeq_count/SRR12845350_transcripts.tsv, results/Rawdata/HTSeq_count/SRR8094770_transcripts.tsv
    output: results/Counts/9606_genes.tsv, results/Counts/9606_transcripts.tsv
    jobid: 1

[Wed Nov 11 19:39:03 2020]
Error in rule decrease_zero_inflation:
    jobid: 1
    output: results/Counts/9606_genes.tsv, results/Counts/9606_transcripts.tsv
    conda-env: /home/kasper/Git_repositories/LiverDB/.snakemake/conda/9e8b782a

RuleException:
URLError in line 249 of /home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py:
<urlopen error [Errno 21] Is a directory: '/home/kasper/Git_repositories/LiverDB/workflow/scripts'>
  File "/home/kasper/Git_repositories/LiverDB/workflow/Snakefile.py", line 249, in __rule_decrease_zero_inflation
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 222, in urlopen
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 525, in open
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 543, in _open
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 503, in _call_chain
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 1453, in file_open
  File "/home/kasper/miniconda3/lib/python3.7/urllib/request.py", line 1492, in open_local_file
  File "/home/kasper/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/kasper/Git_repositories/LiverDB/workflow/.snakemake/log/2020-11-11T193903.564442.snakemake.log
